sunpy/net/helio/tests/test_helio.py

```diff
@@ -319,5 +319,5 @@ def test_HECResponse_iter(client):
     table_name = 'rhessi_hxr_flare'
     res = client.search(a.Time(start, end), a.helio.TableName(table_name), a.helio.MaxRecords(10))
     for i in res:
-        # Just to make sure iter still works.
-        assert len(i) == 1
+        # Just to make sure iter still works, check number of columns
+        assert len(i) == 13
```

Comment:
Is this a bugfix? (perhaps the same I asked about above?)

Reply:
No this is a fundamental behaviour change in the response object. When you slice or iterate over the response object now you get back a row object not a length one version of a query response object.